console/create_app/resources/home_resource.ts

```diff
@@ -0,0 +1,25 @@
+import { Drash } from "../deps.ts";
+
+export default class HomeResource extends Drash.Http.Resource {
+    static paths = ["/"];
+
+    public GET() {
+        this.response.body = this.response.render('/index.html');
```

Comment:
do you think we should make another `home_resource.ts` file for the `--api` option? Otherwise, this will respond with `false` when we go to `localhost:1667`

Reply:
Yeah, it makes sense, added here: https://github.com/drashland/deno-drash/pull/217/commits/4d04ccfc3c9a8247eb5a7f3d43717a76fa9693f1